routers/api/v1/repo/fork.go

```diff
@@ -141,7 +141,10 @@ func CreateFork(ctx *context.APIContext) {
 		Description: repo.Description,
 	})
 	if err != nil {
-		if repo_service.IsErrForkAlreadyExist(err) || repo_model.IsErrRepoAlreadyExist(err) || repo_model.IsErrReachLimitOfRepo(err) {
+		if repo_service.IsErrForkAlreadyExist(err) ||
+			repo_model.IsErrRepoAlreadyExist(err) ||
+			repo_model.IsErrRepoFilesAlreadyExist(err) ||
+			repo_model.IsErrReachLimitOfRepo(err) {
```

Comment:
Can't this be shortened to `errors.Is(err, util.IsErrAlreadyExist)`?

Reply:
Done.